charts/aws-efs-csi-driver/values.yaml

```diff
@@ -94,6 +100,11 @@ controller:
     runAsUser: 0
     runAsGroup: 0
     fsGroup: 0
+  # securityContext on the controller container
+  containerSecurityContext:
+    privileged: true
+  leaderElectionRenewDeadline: 10s
```

Comment:
For other reviewers, this `leaderElectionRenewDeadline` and `leaderElectionLeaseDuration` are the default values. [See this readme for more info](https://github.com/kubernetes-csi/external-provisioner)